CovertInfiltration/Src/CovertInfiltration/Classes/CI_DataStructures.uc

```diff
@@ -19,4 +19,12 @@ struct ActionFlatRiskSitRep
 {
 	var name FlatRiskName;
 	var name SitRepName;
+};
+
+struct DelayedReinforcementSpawner
```

Comment:
it's not really `Spawner`. A better name would be `DelayedReinforcementsOrder` or something like that